src/Token/Parser.php

```diff
@@ -39,10 +46,26 @@ public function parse(string $jwt): TokenInterface
         );
     }
 
+    /**
+     * Parse the given JWT string into a plain token.
+     *
+     * @throws InvalidTokenStructure if the input unexpectedly decodes to an encrypted token.
+     */
+    public function parseUnencrypted(string $unencrypted): UnencryptedToken
```

Comment:
I don't know how to test this, other than repeating the tests for `parse()` - perhaps they should be parameterized with a data provider to test both `parse()` and `parseUnencrypted()`?